styles.css

```diff
@@ -95,13 +97,14 @@ hr {
 }
 
 .mobile-menu {
-  display: none;
+  display: flex;
   flex-direction: column;
   width: 100%;
   height: 100vh;
   border: 2px solid rgba(255, 255, 255, 0.78);
   z-index: 2;
   position: fixed;
+  left: 100%;
   background-color: rgb(235, 241, 239);
 }
```

Comment:
- [ ] Hello @godybuoy, great effort working on the changes requested by the previous reviewer, But i notice you only implement just little part of the changes, Please i would kindly advice you stick to the behance design as strict as possible, because this will show the confidence in your ability to replicate any given design.
I have recorded a video on where you still need to pay attention to. Kindly rectify it before re-submitting for review.

Video 👇 


https://user-images.githubusercontent.com/78634841/235329309-a0c6fc5e-a86a-43e2-b8ad-7bcdcc161b88.mp4